README.md

```diff
@@ -73,6 +73,15 @@ To learn more about the platform, check out our [documentation](https://github.c
   ./scripts/install_platform.sh
   ```
 
+- Natively (using sc-component-manager)
+
+  ```sh
+  git clone https://github.com/ostis-ai/ostis-web-platform
+  cd ostis-web-platform/scripts
+  ./install_platform_with_component_manager.sh
+  ./run_sc_component_manager.sh -c ../ostis-web-platform.ini
```

Comment:
Why you need to pass this config? It passed in venvs

Reply:
![image](https://github.com/ostis-ai/ostis-web-platform/assets/36566977/a0888844-961d-4e05-bca5-aa6c9e21e7bd)

Reply:
By default, sc-server provide CONFIG_PATH from venvs. Can we make this for sc-component-manager?